eng/pipelines/runtime.yml

```diff
@@ -60,1445 +60,3 @@ extends:
       #
       - ${{ if eq(variables.dependOnEvaluatePaths, true) }}:
         - template: /eng/pipelines/common/evaluate-default-paths.yml
-
```

Comment:
Why are we deleting all this?

Reply:
This was just a quick-and-dirty way of disabling the runtime pipeline so it doesn't run every time I push to this PR. I'll make sure to restore `runtime.yml` before removing the NO-MERGE label.

Reply:
Oh okok that's fair.